deploy/future

```diff
@@ -0,0 +1,172 @@
+check_py simple_tag # warning
+check_py page_range
+check_py REASON_PHRASES
+check_py STATUS_CODE_TEXT
+check_py ValuesQuerySet
+#check_py ValuesListQuerySet
+#check_py get_context_data
+check_py SimpleTestCase # see also 1.10
+check_py app_name
+check_py total_ordering
+#check_html url # warning
+check_py enable_comments
+check_py setup_databases # warning
+check_py teardown_databases # warning
+check_py models.fields.related
+
+# Deprecated, removed in 1.9
+# (if we had logging for deprecation warnings, we wouldn't need these)
+# a lot of the commented out checks are just false positives
+check_py dictconfig
+check_py importlib
+check_py tzinfo
+#check_py utils.unittest
+#check_py syncdb # also catches pre_syncdb and post_syncdb
+check_py IPAddressField
+check_py handle_app
+check_py RequestSite
+check_py get_current_site
+check_py runfcgi
+check_py SortedDict
+check_py declared_fieldsets
+check_py django.contrib.admin.util
+check_py django.contrib.gis.db.backends.util
+check_py django.db.backends.util
+#check_py django.forms.util
+check_py get_formsets
+check_py _get_memcache_timeout
+check_py dumpdata # warning
+check_py use_natural_keys
+check_py get_declared_fields
+check_py SplitDateTimeWidget
+#check_py REQUEST
+check_py MergeDict
+#check_py memoize
+#check_py get_cache
+#check_py django.db.models.loading
+#check_py QuerySet # warning
+check_py requires_model_validation
+check_py validator_class
+#check_py validate
+check_py validate_field
+check_py import_by_path
+#check_html future # see also 1.10
+check_py javascript_quote
+#check_py TEST_
+check_py cache_choices
+#check_py RedirectView # warning
+check_py FlatPageSitemap
+check_py TestTemplateLoader
+check_py contenttypes.generic
+
+# Deprecated, removed in 1.10
+check_py SQLCompiler
+#check_py patterns
+#check_py reverse
+#check_py optparse
+#check_py NoArgsCommand
+#check_py context_processors
+#check_py aggregates
+check_py aggregate_select
+check_py add_aggregate
+check_py set_aggregate_mask
+check_py append_aggregate_mask
+#check_py resolve_variable
+#check_py get_field_by_name
+#check_py get_all_field_names
+check_py get_fields_with_model
+check_py get_concrete_fields_with_model
+check_py get_m2m_with_model
+#check_py get_all_related_objects
+#check_py get_all_related_many
+check_py get_all_related_m2m
+#check_py error_message
+check_py unordered_list
+check_py _has_changed
+check_py removetags
+check_py remove_tags
+check_py strip_entities
+check_py is_admin_site
+#check_py SubfieldBase
+check_py checksums
+check_py original_content_type_id
+check_py FormMixin
+check_py ALLOWED_INCLUDE_ROOTS
+#check_py TEMPLATE_
+#check_py BaseLoader
+check_py current_app
+#check_py dictionary
+#check_py context_instance
+#check_py dirs
+check_py "\.related"
+#check_py "\-\-list"
+#check_html ssi
+#check_py Storage
+#check_py success_url
+check_py GeoQuerySet
+#check_py ContentType
+check_py allow_migrate
+#check_html cycle
+check_py Signer
+
+if [ $EXIT = 0 ]; then
+    echo "success!"
```

Comment:
If you want you can steal the code from `deploy/lint` to make this print on the same line if there was no output.